bases/br_sp_seduc_fluxo_escolar/escola/publish.sql

```diff
@@ -0,0 +1,41 @@
+/*
+
+Query para publicar a tabela.
+
+Esse é o lugar para:
+    - modificar nomes, ordem e tipos de colunas
+    - dar join com outras tabelas
+    - criar colunas extras (e.g. logs, proporções, etc.)
+
+Qualquer coluna definida aqui deve também existir em `table_config.yaml`.
+
+# Além disso, sinta-se à vontade para alterar alguns nomes obscuros
+# para algo um pouco mais explícito.
+
+TIPOS:
+    - Para modificar tipos de colunas, basta substituir STRING por outro tipo válido.
+    - Exemplo: `SAFE_CAST(column_name AS NUMERIC) column_name`
+    - Mais detalhes: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+
+*/
+
+CREATE VIEW basedosdados-dev.br_sp_seduc_fluxo_escolar.escola AS
+SELECT 
+SAFE_CAST(ano AS STRING) ano,
+SAFE_CAST(municipio AS STRING) municipio,
+SAFE_CAST(diretoria AS STRING) diretoria,
+SAFE_CAST(codigo_rede_ensino AS STRING) codigo_rede_ensino,
+SAFE_CAST(codigo_tipo_escola AS STRING) codigo_tipo_escola,
```

Comment:
Variável tá toda nula. É isso mesmo?

Reply:
sim, to só mantendo igual a base da seduc! na verdade a base deles tem umas situaçõezinhas chatas

Reply:
Explica mais um pouco?